packages/shared/src/hooks/usePlusSubscription.ts

```diff
@@ -30,8 +36,29 @@ export const usePlusSubscription = (): {
     [logEvent],
   );
 
+  const plusHref = useMemo(() => {
+    if (!isPlus) {
+      return plusUrl;
+    }
+
+    // Plus users with Apple StoreKit on iOS get no URL (handled by onClick)
+    if (isIOSNative() && plusProvider === SubscriptionProvider.AppleStoreKit) {
+      return undefined;
+    }
+
+    // External subscription management for iOS and Paddle users
+    if (isIOSNative() || plusProvider === SubscriptionProvider.Paddle) {
+      return managePlusUrl;
+    }
```

Comment:
Should it just be if `plusProvider` is paddle show the `managePlusUrl`?

Reply:
You're right 🙈 

Hopefully Apple lets us open the external manage link when plus is managed via Paddle